compiler/src/builtin_functions.rs

```diff
@@ -4,17 +4,40 @@ use strum_macros::EnumIter;
 
 #[derive(Debug, EnumIter, PartialEq, Eq, Clone, Hash, Copy)]
 pub enum BuiltinFunction {
-    Add,              // (summandA: int) (summandB: int) -> (sum: int)
-    Equals,           // (a: any) (b: any) -> (boolean: True | False)
-    GetArgumentCount, // (closure: Lambda) -> (argumentCount: int)
-    IfElse, // (condition: True | False) (then: lambda) (else: lambda) -> (resultOfExecutedClosure: any)
-    Print,  // (message: Text) -> Nothing
-    StructGet, // (struct: struct) (key: any) -> (value: any)
-    StructGetKeys, // (struct: struct) -> (listOfKeys: listOfAny)
-    StructHasKey, // (struct: struct) (key: any) -> (isKeyInStruct: True | False)
-    TypeOf, // (value: any) -> (type: Int | Text | Symbol | Struct | Function | Builtin)
-    UseAsset, // (currentPath: struct) (target: text) -> (target: text)
-    UseLocalModule, // (currentPath: struct) (target: text) -> (target: struct)
+    Call,                // (lambdaWith0Arguments) -> (returnValue: any)
+    Equals,              // any any -> booleanSymbol
+    GetArgumentCount,    // closure -> argumentCount
+    IfElse,              // condition thenClosure elseClosure -> resultOfExecutedClosure
+    IntAdd,              // (summandA: int) (summandB: int) -> (sum: int)
+    IntBitLength,        // (value: int) -> (numberOfBits: int)
+    IntBitwiseAnd,       // (valueA: int) (valueB: int) -> (result: int)
+    IntBitwiseOr,        // (valueA: int) (valueB: int) -> (result: int)
+    IntBitwiseXor,       // (valueA: int) (valueB: int) -> (result: int)
+    IntCompareTo,        // (valueA: int) (valueB: int) -> (ordering: Less | Equal | Greater)
+    IntDivideTruncating, // (dividend: int) (divisor: int) -> (quotient: int)
+    IntModulo,           // (dividend: int) (divisor: int) -> (remainder: int)
+    IntMultiply,         // (factorA: int) (factorB: int) -> (product: int)
+    IntParse,            // (text: text) -> (parsedInt: maybeOfInt)
```

Comment:
I'm not entirely sure when the arguments in the comments have types and when not, but this looks like a very obvious candidate for omitting the type.

```suggestion
    IntParse,            // text -> (parsedInt: maybeOfInt)
```